app/decorators/contribution_decorator.rb

```diff
@@ -0,0 +1,20 @@
+class ContributionDecorator < Draper::Decorator
+  delegate_all
+
+  def user_short_name
+    user.name.split.values_at(0, -1).uniq.join(' ')
+  end
+
+  def created_at
+    model.created_at.strftime("%d/%m/%Y")
+  end
+
+  def reviewed_by_short_name
+    reviewed_by.name.empty? ? '' : reviewed_by.name.split.values_at(0, -1).uniq.map(&:first).join.upcase
```

Comment:
It seems like `User#name` is always required to be present even though the `reviewed_by` itself is optional on the Contribution model. Maybe what you want is only `reviewed_by&.name.split.values_at(0, -1).uniq.map(&:first).join.upcase`?